kmir/src/kmir/__main__.py

```diff
@@ -51,11 +51,13 @@ def exec_version(**kwargs: Any) -> None:
     print(f'KMIR Version: {VERSION}')
 
 
-def exec_init(llvm_dir: str, **kwargs: Any) -> KMIR:
+def exec_init(llvm_dir: str, **kwargs: Any) -> None:
     print(
-        'WARN: "init" was seen in args, this calls an internal function. If a file is named "init", it must be renamed'
+        'kmir init: Building GLR parser'
```

Comment:
I think the warning is actually not necessary, and it shows some confusion on my part. Just a log to say we are building the parser is fine.